tailwind.config.js

```diff
@@ -0,0 +1,198 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}', './data/**/*.{js,ts,jsx,tsx}'],
+  safelist: [
+    'bg-[#01a1df]',
```

Comment:
I am not sure why we still have these magic digits here, can we convert them to the closest colors our color list, please

Reply:
noted